crates/ruff_python_parser/src/string.rs

```diff
@@ -294,7 +301,7 @@ impl StringParser {
                     if !ch.is_ascii() {
                         return Err(LexicalError::new(
                             LexicalErrorType::InvalidByteLiteral,
-                            self.range(self.get_pos()),
+                            TextRange::empty(self.position()),
```

Comment:
Correct me if I'm wrong but I don't think this branch is even reachable because the `memchr3` call only tries to find either `{`, `}`, or `\` which the match statement covers up.

Reply:
Ok, so this code _is_ reachable but still the `ch` will always be `\` because the previous match case is something like:

```rs
	'\\' if !self.kind.is_raw_string() && self.peek_byte().is_some() => ...
	ch => ...
```

I can confirm that by running the entire test suite.